modules/installation-and-upgrade/pages/container-deployment/suma/server-deployment-suma.adoc

```diff
@@ -174,17 +174,6 @@ transactional-update
 
 . Reboot.
 . Log in as root.
-. Install the container utilities:
```

Comment:
I don't think we need the previous line, 176?

Reply:
Removed